frontend/cypress/integration/common/overview.ts

```diff
@@ -23,11 +23,11 @@ Before(() => {
 
 Given('user opens the overview page', () => {
     // Forcing "Pause" to not cause unhandled promises from the browser when cypress is testing
-    cy.visit(url + '/overview?refresh=0&duration=60');
+    cy.visit(url + '/overview?refresh=0');
```

Comment:
I don't think this is 100% good, as part of the test is to click the time selector button to change the duration.